src/Knp/Component/Pager/Paginator.php

```diff
@@ -96,8 +97,9 @@ public function setDefaultPaginatorOptions(array $options): void
      * @throws PageNumberOutOfRangeException
      * @return PaginationInterface
      */
-    public function paginate($target, int $page = 1, int $limit = 10, array $options = []): PaginationInterface
+    public function paginate($target, int $page = 1, int $limit = null, array $options = []): PaginationInterface
```

Comment:
Can't we just use `self::DEFAULT_LIMIT` as default value?

Reply:
The idea of a change was to make default limit value configurable in the bundle config. 
Using just constant as a default value, not value from the options, will make it impossible to overwrite value from the bundle configuration.